src/createServer.js

```diff
@@ -1,13 +1,20 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { userRouter } = require('./routers/user.router');
+const { expenseRouter } = require('./routers/expense.router');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
```

Comment:
You can create 2 functions that will reset your data arrays.
`setInitialUsers()` should change the users array to an empty array;
`setInitialExpenses()` should change the expenses array to an empty array;

Save this function in their services and call in the `createServer()` function.

It will help you pass the tests